delira/training/callbacks/abstract_callback.py

```diff
@@ -59,3 +59,41 @@ def at_epoch_end(self, trainer, **kwargs):
 
         """
         return {}
+
+    def at_training_begin(self, trainer, **kwargs):
```

Comment:
I think we should explicitly add the arguments.

Reply:
As stated in the other PR ( #172 ) this might be troublesome, since these callbacks are designed to work with every trainer/backend/callback implementation, which might require different arguments here. We would have to discuss what to add and what not to add first

Reply:
Closing this for the discussion in the other PR.